<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet Vision</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(201, 218, 220)</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1151</width>
     <height>611</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="encodage">
    <attribute name="title">
     <string>Encodage</string>
    </attribute>
    <widget class="QTextEdit" name="secretText_encodage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Entrer votre texte secret</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>150</y>
       <width>511</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="plot_img_encodage"/>
    </widget>
    <widget class="QPushButton" name="btn_import_encodage">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

  background-color: rgb(138, 166, 180);
  border-radius: 8px;
  border-width: 0;
  color: rgb(255, 255, 255);
  font-size: 13px;
  height: 52px;
  line-height: 1;
  margin: 0;
  outline: none;
  padding: 0 32px;
  text-align: center;
  text-decoration: none;
  vertical-align: baseline;
}

</string>
     </property>
     <property name="text">
      <string>Importer image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_encodage">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

  background-color: rgb(138, 166, 180);
  border-radius: 8px;
  border-width: 0;
  color: rgb(255, 255, 255);

  font-size: 13px;
  height: 52px;
  line-height: 1;
  margin: 0;
  outline: none;
  padding: 0 32px;
  text-align: center;
  text-decoration: none;
  vertical-align: baseline;
  white-space: nowrap;
}


</string>
     </property>
     <property name="text">
      <string>Encoder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_enregistrer">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>80</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

  background-color: rgb(138, 166, 180);
  border-radius: 8px;
  border-width: 0;

  color: rgb(255, 255, 255);



  font-size: 13px;
  height: 52px;

  line-height: 1;
  margin: 0;
  outline: none;
  padding: 0 32px;
  text-align: center;
  text-decoration: none;
  vertical-align: baseline;
  white-space: nowrap;

}


</string>
     </property>
     <property name="text">
      <string>Enregistrer</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>511</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="plot_img_text_encode"/>
    </widget>
    <widget class="QPushButton" name="btn_create_img_text">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

  background-color: rgb(138, 166, 180);
  border-radius: 8px;
  border-width: 0;
  color: rgb(255, 255, 255);



  font-size: 13px;
  height: 52px;
  line-height: 1;
  margin: 0;
  outline: none;
  padding: 0 32px;
  text-align: center;
  text-decoration: none;
  vertical-align: baseline;
  white-space: nowrap;

}

</string>
     </property>
     <property name="text">
      <string>Cr√©er l'image</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="decodage">
    <attribute name="title">
     <string>Decodage</string>
    </attribute>
    <widget class="QPushButton" name="btn_import_decodage">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

  background-color: rgb(138, 166, 180);
  border-radius: 8px;
  border-width: 0;

  color: rgb(255, 255, 255);



  font-size: 13px;
  height: 52px;
 
  line-height: 1;
  margin: 0;
  outline: none;
  padding: 0 32px;
  text-align: center;
  text-decoration: none;
  vertical-align: baseline;
  white-space: nowrap;
}


</string>
     </property>
     <property name="text">
      <string>Importer image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_decodage">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  background-color: rgb(138, 166, 180);
  border-radius: 8px;
  border-width: 0;

  color: rgb(255, 255, 255);
 
  font-size: 13px;
  height: 52px;
  
  line-height: 1;
  margin: 0;
  outline: none;
  padding: 0 32px;
  text-align: center;
  text-decoration: none;
  vertical-align: baseline;
  white-space: nowrap;

}


</string>
     </property>
     <property name="text">
      <string>Decoder</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>511</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="plot_img_decodage"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>120</y>
       <width>511</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="plot_img_text_decod"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
